tests/_test_base.bash

```diff
@@ -66,9 +66,13 @@ function test_user_password {
 function stop_gpg_agent {
   local username
   username=$(id -u -n)
-  ps -wx -U "$username" | gawk \
-    '/gpg-agent --homedir/ { if ( $0 !~ "awk" ) { system("kill "$1) } }' \
-    > /dev/null 2>&1
+  local pid
+  pid=$(pgrep -U "$username" -x gpg-agent)
```

Comment:
Is it important that I preserve the `--homedir` portion of the regex?

Reply:
Without it `make test` may kill **gpg-agent** for the current user.

Reply:
That is true. Technically it could still kill the wrong gpg-agent(s), if the user had used `--homedir`. Let me see if I can't improve things a little with `pgrep -f`, which matches against the whole command line

Reply:
Here's what I'm using now, I think it's pretty much guaranteed not to match the user's `gpg-agent`: 

`pgrep -U "$username" -f "gpg-agent --homedir.*$TEST_GPG_HOMEDIR"`

Reply:
let me know if you have any thoughts @Mohjive